docs/reference/intro.asciidoc

```diff
@@ -155,81 +184,146 @@ You can use a combination of dynamic and explicit mapping on the same index.
 This is useful when you have a mix of known and unknown fields in your data.
 ====
 
+// New html page
+[[es-ingestion-overview]]
+=== Add data to {es}
```

Comment:
I'd like to see this topic get some more content from its big brother in the cloud docs. we also don't talk about pipelines, data manip, etc. which are foundational concepts

Reply:
I don't know the ingest story super well, but I can see understanding that this is the point at which you can enrich data to customize it to your use case is really a production-side concern (esp for search use case). I view prod concerns as scaling, security, UAC ...

this is doing a lot of heavy lifting but I do think exposing those things here aligns with the 101 ask